src/Client/Grpc/GrpcDurableTaskClient.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) Microsoft Corporation.
+// Copyright (c) Microsoft Corporation.

```

Comment:
Do you know why this is showing up in the diff?

Reply:
Unfortunately no, it doesn't seem like a line ending thing so I'm at a loss as to how to get rid of it...

Reply:
Ok, no worries. It should be fine.